devices/ace3dsplus.cpp

```diff
@@ -1,3 +1,5 @@
+#ifdef TESTING
```

Comment:
this ifdef could be easily moved to ~line 586 (a similar change could be made to r4igoldcc.cpp as well); the "Ace3DSPlus ace3DSplus;" line is what adds it to the list of flashcarts, so it's way overkill to wrap the whole thing in ifdefs. In any case, I hate ifdefs, personally... in most cases anyway.

Reply:
I guess the difference is that this doesn't even have to compile and definately won't add any bloat to the executable if the whole thing removed, while it still has to compile (which honestly I find a *good* thing, makes it harder to accidentally make breaking changes) if we only remove that one line.